common/app/views/fragments/javaScriptFirstSteps.scala.html

```diff
@@ -194,9 +194,19 @@
 
 </script>
 
-@* Not the usual type of thing we do, but I want this separate to our "normal" javascript. *@
-@* also delete iPhoneConfidenceCheck inside of facia.js *@
+
+<script>
+    guardian.isIphone6 = false;
+    guardian.isIphone4 = false;
+    guardian.inTestBucket = Math.floor(Math.random() * 2 + 1) === 1;
```

Comment:
I know that it's been merged already, but this line is quite hard to read.

For the future i'd suggest something like
`guardian.inTestBucket = Math.random() < 0.5;`
It's easier to understand the ratio